src/ONE.Models.CSharp/ONE.Models.CSharp.csproj

```diff
@@ -2,9 +2,9 @@
 
   <PropertyGroup>
     <!--For Release packages update the VersionPrefix and remove any VersionSuffix-->
-    <VersionPrefix>4.12.0</VersionPrefix>
+    <VersionPrefix>4.13.0</VersionPrefix>
     <!--For Feature packages update the VersionSuffix with alpha and an incremental number. E.g. alpha1, alpha2...-->
-    <VersionSuffix></VersionSuffix>
+    <VersionSuffix>pre-1</VersionSuffix>
```

Comment:
are we supposed to have suffix for develop branch release?

Reply:
No, sometimes I remove it before code review and sometimes after.

Reply:
For this repo I usually wait because I like to merge the protocolBuffers first and then update the submodule to the develop branch before publishing a package.

Reply:
I think we need to communicate the process for pre-release versions. Just noticed that there is a pre-release for shared.config in the develop branch.

Reply:
Here is a Decision Log Jon created, not sure if it was converted to a SW page or not. https://aquaticinformatics.atlassian.net/wiki/spaces/RIO/pages/157909029/Decision+NuGet+Package+Release+Strategy

Reply:
Lol, just saw the SW page was linked on that decision log, d'oh!

Reply:
Yep, there's SW for this :)

Reply:
I think I'd rather just abandon master and have develop in this case.